schemas/expconf/v0/log-policy.json

```diff
@@ -3,22 +3,32 @@
     "$id": "http://determined.ai/schemas/expconf/v0/log-policy.json",
     "title": "LogPolicy",
     "additionalProperties": false,
-    "required": [
-        "pattern",
-        "action"
-    ],
     "type": "object",
     "properties": {
+        "name": {
+            "type": [
+                "string",
+                "null"
+            ],
+            "$comment": "Comment can be found in type LogPolicyV0",
```

Comment:
I don't understand what `Comment can be found in type LogPolicyV0` means?

Reply:
Add actual comment here instead of linking files.